notes/notesRoutes.js

```diff
@@ -0,0 +1,101 @@
+const express = require("express");
+
+const notes = require("./notesModel.js");
+
+const router = express.Router();
+
+//Get a list of all notes
+router.get("/", (req, res) => {
+  notes
+    .find()
+    .then(notes => {
+      res.status(200).json(notes);
+    })
+    .catch(err => res.status(500).json(err));
```

Comment:
Write out your own descriptive error so you know where the problem occurred. Something like "database could not GET note".